optuna/samplers/_tpe/parzen_estimator.py

```diff
@@ -40,11 +40,10 @@ def __init__(
         parameters: _ParzenEstimatorParameters,
         predetermined_weights: np.ndarray | None = None,
     ) -> None:
```

Comment:
What about adding this line because this class should not be touched by users anyways?

```python
if not parameters.consider_prior:
    raise ValueError("The implementation for `consider_prior=False` has been removed at v4.3.0.")
```